qiskit_sphinx_theme/furo/base/static/styles/qiskit_changes.css

```diff
@@ -1,23 +1,37 @@
-/* Define some Furo variables. See
```

Comment:
I reorganized how we do comments in this file. This is partially prework for using Carbon icons, which is going to override 10+ variables. The `body` rules were getting too hard to understand.